client/core/trade.go

```diff
@@ -1047,32 +1062,47 @@ func (t *trackedTrade) isSwappable(ctx context.Context, match *matchTracker) boo
 	// Just a quick check here. We'll perform a more thorough check if there are
 	// actually swappables.
 	if !wallet.locallyUnlocked() {
-		t.dc.log.Errorf("not checking if order %s, match %s is swappable because %s wallet is not unlocked",
+		t.dc.log.Errorf("Order %s, match %s is not swappable because %s wallet is not unlocked",
 			t.ID(), match, unbip(wallet.AssetID))
 		return false
 	}
 
-	if match.Status == order.MakerSwapCast {
+	switch match.Status {
+	case order.NewlyMatched:
+		return match.Side == order.Maker
+	case order.MakerSwapCast:
 		// Get the confirmation count on the maker's coin.
 		if match.Side == order.Taker {
```

Comment:
While we're improving here, maybe  a 

```Go
if match.Side == order.Maker {
    return false
}
```

So we can lose a level of indentation?